internal/kafka/internal/presenters/cluster.go

```diff
@@ -70,22 +68,12 @@ func PresentEnterpriseCluster(cluster api.Cluster, consumedStreamingUnitsInTheCl
 		SupportedInstanceTypes:        public.SupportedKafkaInstanceTypesList{},
 	}
 
-	// enterprise clusters only supports standard instance type for now. It is safe to hardcode this.
-	storedCapacityInfo, ok := cluster.RetrieveDynamicCapacityInfo()[types.STANDARD.String()]
-	if ok {
-		presentedCluster.CapacityInformation = presentEnterpriseClusterCapacityInfo(consumedStreamingUnitsInTheCluster, storedCapacityInfo)
```

Comment:
This logic has been removed but the type still presents the capacity information as an attribute from the response right?
Is this what we want?

Reply:
Why has this line `presentedCluster.CapacityInformation = presentEnterpriseClusterCapacityInfo(consumedStreamingUnitsInTheCluster, storedCapacityInfo)` been removed?